backend/src/main/kotlin/outcobra/server/util/DateOverlabUtil.kt

```diff
@@ -0,0 +1,21 @@
+package outcobra.server.util
```

Comment:
|`line 8`|
|---|
You could create an extension function for SchoolYear that utilizes this internally for a nicer usage (`schoolYearA.overlaps(schoolYearB)`